.travis/test.sh

```diff
@@ -18,6 +18,8 @@ then
   UT_FLAGS+=" -K combined_modes"
 fi
 
+UT_FLAGS+=" -K mock_read_routes6_windows "
```

Comment:
Good idea, where should we put the .uts ? I guess as it's part of the main scapy in test/ ?